funnel/models/space.py

```diff
@@ -180,6 +186,13 @@ def permissions(self, user, inherited=None):
                     'edit-schedule',
                     'new-session',
                     'edit-session',
+                    'event-checkin',
+                    'event-view',
+                    'ticket-type-view',
+                    'event-sync',
+                    'participant-edit',
+                    'participant-view',
+                    'new-participant'
```

Comment:
We'll need to be consistent with `verb-noun` or `noun-verb` order with permissions. Our convention appears to be `verb-noun` for permission names but `noun-verb` for view names (which in itself is a confusing difference).